clang/lib/Headers/mm3dnow.h

```diff
@@ -7,151 +7,16 @@
  *===-----------------------------------------------------------------------===
  */
 
+// 3dNow intrinsics are no longer supported, and this header remains only as a
+// stub for users who were including it to get to _m_prefetch or
+// _m_prefetchw. Such uses should prefer x86intrin.h.
```

Comment:
How best can we add a deprecation warning to builds for people who are doing this?

Reply:
Looking at this again, I do think it's reasonable to emit a `#warning` here.

I was initially worried that might cause issues for users, but there's so very very few users of anything 3dnow-related to start with, and the ones I found generally conditioned the include on `__3dNOW__` being defined, anyhow.